Contentstack.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
 s.name             = 'Contentstack'
-s.version          = '3.10.1'
+s.version          = '3.11.0'
 s.summary          = 'Contentstack is a headless CMS with an API-first approach that puts content at the centre.'
```

Comment:
Contentstack is an API-first CMS platform. This essentially means that it is Content Delivery SDK to get anything you wish to get from delivery API. @uttamukkoji 
I am checking it from docs team